utils/package.json

```diff
@@ -45,13 +45,13 @@
     "graphql-markdown": "^6.0.0",
     "jest": "^27.5.1",
     "js-yaml": "^4.1.0",
+    "neatjson": "^0.10.6",
```

Comment:
It looks a fair bit of the CI checks are failing because this new dependency is not in the yarn.lock file. 

```
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`
```

Lets make sure this dependency was installed through yarn